plugins/orchestrator/src/components/WorkflowDialog/WorkflowDialog.tsx

```diff
@@ -55,9 +60,16 @@ export const WorkflowDialog = (
       </DialogTitle>
       <DialogContent>
         <Box className={classes.editor}>
-          <WorkflowEditor {...props} workflowId={workflowId} />
+          <WorkflowEditor
+            {...props}
+            workflowId={workflowId}
+            ref={forwardedRef}
+          />
         </Box>
       </DialogContent>
+      {props.dialogActions}
     </Dialog>
   );
 };
+
+export const WorkflowDialog = forwardRef(RefForwardingWorkflowDialog);
```

Comment:
Are you using this ref?

Reply:
https://github.com/caponetto/backstage-plugins/pull/59/files#diff-39da7eb41123e248a4fd6670e4e3ee395e95dce50cf17a76de47ebbfec551e4fR82